scripts/multipart_upload_via_API.py

```diff
@@ -0,0 +1,142 @@
+import os
+import hashlib
+from base64 import b64encode
+import argparse
+from hashlib import md5
+
+import requests
+import multihash
+
+
+def b64_md5(data):
+    '''Return the Base64 encoded MD5 digest of the given data
+
+    Args:
+        data: string
+            input data
+
+    Returns:
+        string - base64 encoded MD5 digest of data
+    '''
+    return b64encode(md5(data).digest()).decode('utf-8')
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description="Utility script for uploading large asset files as a multipart upload " \
+            "using the STAC API. When calling this script, summon can be used to inject secrets " \
+                "from gopass into the environment variables. To do this, call this script like " \
+                    "this: summon -p gopass -e dev|int|prod python3 multipart_upload_via_API.py " \
+                        "localhost|DEV|INT|PROD collection-name item-name assset-name " \
+                            "asset-file-path --part-size."
```

Comment:
I'll move the description on how to use summon from this description here into the `README.md`